cluster/charts/rook-ceph-cluster/values.yaml

```diff
@@ -5,6 +5,9 @@
 # Namespace of the main rook operator
 operatorNamespace: rook-ceph
 
+# The metadata.name of the CephCluster CR. The default name is the same as the namespace.
+#clusterName: rook-ceph
```

Comment:
nit
```suggestion
# clusterName: rook-ceph
```

Reply:
It's consistent with other settings that are commented out in this values.yaml, as well as other manifests such as cluster.yaml. It's just a tiny thing and does look awkward at first glance, but I do think it helps differentiate between a real comment and a commented-out setting. Maybe there is a better pattern for another PR though...